bindata/bootkube/bootstrap-manifests/etcd-member-pod.yaml

```diff
@@ -43,41 +43,38 @@ spec:
 
       source /run/etcd/environment
 
-      # TODO remove when this is handeled in setup-etcd-env
-      ETCD_IP_ADDRESS=${ETCD_IPV4_ADDRESS:-$ETCD_IPV6_ADDRESS}
-
-      [ -e /etc/ssl/etcd/system:etcd-server:${ETCD_IP_ADDRESS}.crt -a \
-        -e /etc/ssl/etcd/system:etcd-server:${ETCD_IP_ADDRESS}.key ] || \
+      [ -e /etc/ssl/etcd/system:etcd-server:${ETCD_ESCAPED_IP_ADDRESS}.crt -a \
```

Comment:
Why are we using IP addresses?  Did it just seem convenient for uniqueness?  How about node name instead which eliminates this use of IP addresses?

Migration could have a step that copies a file (if present) to the new name.

Reply:
> Why are we using IP addresses? Did it just seem convenient for uniqueness?

right

> How about node name instead which eliminates this use of IP addresses?

sounds reasonable as this is the bootstrap node. The -e flag is just checking the file exists but it also serves as validation that the TLS certs will match what is expected in SAN. On the bootstrap node this is not a concern.